### PR TITLE
Changelog for 0.38.1

### PR DESCRIPTION
**Pull request by esc**
As Title.

### CODE REVIEW

**Review by gmarkall — APPROVED** [review]
This looks good, the rendered text looks fine in the RTD preview.

File: CHANGE_LOG [hunk 1]
```diff
@@ -1,3 +1,10 @@
+v0.38.1 (May 19, 2022)
+----------------------
+
+This is a maintenance release to support the Apple M1 architecture.
+
+* PR `#841 <https://github.com/numba/llvmlite/pull/841>`_: Merge pull request #813 from seibert/m1_support (`esc <https://github.com/esc>`_, `Stan Seibert <https://github.com/seibert>`_)
```
Comment (stuartarchibald) [inline CHANGE_LOG:6]:
```suggestion
* PR `#841 <https://github.com/numba/llvmlite/pull/841>`_: Merge pull request #813 from seibert/m1_support (`esc <https://github.com/esc>`_, `Stan Seibert <https://github.com/seibert>`_)
* PR `#845 <https://github.com/numba/llvmlite/pull/845>`_: Backport #786 for 0.38.1 (`stuartarchibald <https://github.com/stuartarchibald>`_)
```

**Comment by esc** [comment]
@stuartarchibald thank you for the suggestion, I have accepted them to trigger public CI.

**Review by stuartarchibald — APPROVED** [review]
Thanks for the updates @esc.

**Pull request merged**